### PR TITLE
feat: add polyfill db to documentation

### PR DESCRIPTION
**Pull request by Valerioageno**
## Context & Description

This PR initialize a `polyfill-db.json` file in the documentation in order to keep track of the common missing polyfills

### CODE REVIEW

File: apps/documentation/public/polyfill-db.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+		"polyfills": [
+				{ 
+						"identifier": "HTMLElement",
+						"imports": ["@lit-labs/ssr-dom-shim", "HTMLElement"]
+				},
+				{ 
+						"identifier": "customElements",
+						"imports": ["@lit-labs/ssr-dom-shim", "customElements"]
+				}
+}
```
Comment (marcalexiei) [inline apps/documentation/public/polyfill-db.json]:
```suggestion
				}
		]		
}
```

Comment (marcalexiei) [reply apps/documentation/public/polyfill-db.json]: The json is invalid, the polyfills array is not closed

File: apps/documentation/public/polyfill-db.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+		"polyfills": [
+				{ 
+						"identifier": "HTMLElement",
+						"imports": ["@lit-labs/ssr-dom-shim", "HTMLElement"]
```
Comment (marcalexiei) [inline apps/documentation/public/polyfill-db.json]: The names `imports` suggest a list however this array is referencing to a package name and a named export so IMHO is confusing.

```suggestion
             "import":  {  
						   "package": "@lit-labs/ssr-dom-shim", 
						   "name": "HTMLElement"   
						 }
```

Comment (marcalexiei) [inline apps/documentation/public/polyfill-db.json]: I'm wondering if we should support a list of available polyfills for each element.
Example accounting for my previous comment:

```json
{
  "polyfills": [
    {
      "identifier": "HTMLElement",
      "imports": [
        {
          "package": "@lit-labs/ssr-dom-shim",
          "name": "HTMLElement"
        },
        {
          "package": "@anotherOne",
          "name": "HTMLElement"
        }
      ]
    }
  ]
}

```

**Review by marcalexiei — COMMENTED** [review]
1. Could this json be referenced from somewhere in the documentation to let the user know official polyfills (Overview -> Core concepts)?

----

### Additional change

Now prettier runs on all documentation files

**Review by Valerioageno — COMMENTED** [review]
That"s why I waited to merge 😄Thanks

**Comment by Valerioageno** [comment]
Yeah I thought to create a page in which the user can search for it. For now let's just create the db to keep track of them. In the close future we will also do that!

File: apps/documentation/public/polyfill-db.json (hunk 2 above)
Comment (Valerioageno) [reply apps/documentation/public/polyfill-db.json]: I think it gets hard to maintain. WDYT?

Comment (marcalexiei) [reply apps/documentation/public/polyfill-db.json]: Ok, let's keep a single import for each polyfill.

Comment (Valerioageno) [reply apps/documentation/public/polyfill-db.json]: Yeah. Better to use `import`

**Review by Valerioageno — COMMENTED** [review]
refactor: update 'imports' key to 'import'

**Pull request merged**